proto/minder/v1/minder.proto

```diff
@@ -2683,3 +2683,41 @@ message EvaluationHistoryAlert {
     // the structure and contents are alert specific, and are subject to change.
     string details = 2;
 }
+
+message EntityInstance {
+    // id is the unique identifier of the entity.
+    string id = 1;
+
+    // context is the context in which the entity is evaluated.
+    ContextV2 context = 2;
```

Comment:
Question: how is our `ContextV2` support these days? I see V2 being used in one RPC, do our middlewares support V2 already?

Reply:
They do https://github.com/stacklok/minder/blob/main/internal/controlplane/common.go